CONTRIBUTING.md

```diff
@@ -1,70 +1,103 @@
-## Contributing
+# Contributing
 
 | Component | Build Status |
 | --------- | ------------ |
 | Management Libraries | [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) |
 | Client Libraries | [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/41?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master) |
 
-### Prerequisites:
+# Prerequisites:
   Install VS 2017 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
-
-### To build:
-=======
-#### If you have recently cloned the repo or did a pull from upstream or did git clean -xdf you need to the following before you do anything else in order to build your projects
+  <br/><br/>
+# Data Plane Libraries
+## TO BUILD:
+### Single Service from Command Line
+1. Open VS 2017 Command Prompt
+2. Navigate to service directory e.g. *"sdk\eventhub"*
+3. Invoke `msbuild /t:Build -r` or `dotnet build`
```

Comment:
msbuild /t:Build isn't going to work as they will need to do a restore first. So I would simply remove that as an option.

Reply:
Also this option is only going to work if there is a solution in the service directory.